src/OSPSuite.Presentation/UICommands/ParameterIdentificationFeedbackViewVisibilityUICommand.cs

```diff
@@ -1,31 +1,26 @@
 ﻿using OSPSuite.Core.Domain.ParameterIdentifications;
-using OSPSuite.Core.Events;
+using OSPSuite.Core.Services;
 using OSPSuite.Presentation.Presenters.ParameterIdentifications;
 using OSPSuite.Presentation.Services;
-using OSPSuite.Utility.Events;
 
 namespace OSPSuite.Presentation.UICommands
 {
-   public class ParameterIdentificationFeedbackViewVisibilityUICommand : ObjectUICommand<IParameterIdentificationFeedbackPresenter>, IListener<ParameterIdentificationSelectedEvent>
+   public class ParameterIdentificationFeedbackViewVisibilityUICommand : ObjectUICommand<IParameterIdentificationFeedbackPresenter>
```

Comment:
there is in fact a better UI command for this.  ActveSubjectUIComand. Can you use this insteaD?

Reply:
> ActveSubjectUIComand

I did not find ActveSubjectUIComand but instead I found ActiveObjectUICommand. The problem is that we need the command to implement ObjectUICommand<IParameterIdentificationFeedbackPresenter> but retrieve a ParameterIdentification as the active subject. So if I would extend ActiveObjectUICommand<IParameterIdentificationFeedbackPresenter> the Subject will be of type IParameterIdentificationFeedbackPresenter which is not what I need. To avoid this, I would need to change IParameterIdentificationFeedbackManager so it uses the cache based on the presenter, and not the object. This will imply to make the presenter to have a public getter for the object so we can access the object from the Subject (presenter). Do you like these changes?

Reply:
I don't understand. Why can't you be ActiveObjectUICommand<ParameterIdentificatoin>?

Reply:
It was about making the ParameterIdentification public on the presenter interface (IParameterIdentificationFeedbackPresenter). Sorry the description was not that clear. Check changes on the code for easier understanding.

Reply:
I am sorry but I don't understand how this works. Can we have a call?

Reply:
> I am sorry but I don't understand how this works. Can we have a call?

@msevestre Just call any time